draft-ietf-quic-transport.md

```diff
@@ -4414,11 +4416,11 @@ original_connection_id (0x0000):
   server MUST include the original_connection_id transport parameter if it sent
   a Retry packet.
 
-idle_timeout (0x0001):
+max_idle_timeout (0x0001):
 
-: The idle timeout is a value in milliseconds that is encoded as an integer; see
-  ({{idle-timeout}}).  If this parameter is absent or zero then the idle
-  timeout is disabled.
+: The max idle timeout is a value in milliseconds that is encoded as an integer;
+  see ({{idle-timeout}}).  If this parameter is absent or zero then the idle
+  timeout is disabled unless the peer specifies a max idle timeout.
```

Comment:
Maybe rephrase to: "Idle timeout is disabled when this parameter is absent or zero on both sides"?

Reply:
Thanks!